src/javascript/app/ng-wig/views/ng-wig.html

```diff
@@ -2,7 +2,7 @@
   <ul class="nw-toolbar">
     <li class="nw-toolbar__item" ng-repeat="button in toolbarButtons" >
         <div ng-if="!button.isComplex">
-          <button type="button" class="nw-button" title="{{button.title}}" ng-click="execCommand(button.command)" ng-disabled="editMode">
+          <button type="button" class="nw-button" title="{{button.title}}" ng-click="execCommand(button.command)" ng-class="{ 'nw-button--active': button.isActive }" ng-disabled="editMode">
```

Comment:
you can put method here isActive() 

in this case you don't need `updateButtonsState` and related stuff